CONTRIBUTING.md

```diff
@@ -34,6 +34,12 @@ Now install the development dependencies:
 bundle install
 ```
 
+To update javascript bundle run (add `-w` flag for watch mode):
```

Comment:
Since we now require Node and Yarn for development, can you please include a paragraph with that and links to each on how to install? While it’s obvious to us it won’t be to others so we should help them out.

Reply:
Perhaps we can provide a yarn binstub similar to the one provided with Rails applications, and recommend to run that. Like

```ruby
#!/usr/bin/env ruby

begin
  exec "yarnpkg", *ARGV
rescue Errno::ENOENT
  $stderr.puts "Yarn executable was not detected in the system."
  $stderr.puts "Download Yarn at https://yarnpkg.com/en/docs/install"
  exit 1
end
```

Reply:
👍 New rails apps with webpacker do include a `yarn` binstub so I just copied it over here.
Added some documentation as well. `rollup` is being installed through yarn `devDependencies` but it's still a good thing to mention it indeed. 
Also updated calls to use the binstub.

Reply:
Looks great, thanks!